components/fetch-build-scan-data-cmdline-tool/build.gradle.kts

```diff
@@ -29,11 +35,32 @@ java {
     }
```

Comment:
I think there were also some improvements in swagger that we might be able to benefit from. @sorin-florea and @facewindu will know. Let's treat this build optimization item with lowest priority. It can also be tackled post-release of the build validation scripts.

Reply:
I'll definitely check with them tomorrow, but I compared this with the GE API Sample, and how the code generator is used doesn't appear to have changed very much from when I last looked at this.

Reply:
I think this is fine for now but when we release 2022.2.4 you should update to openapi generator 6.0.0. 
 If you don't need this to be built offline you could take the schema from docs.gradle.com instead of including it in the project, [the same way we do in the sample](https://github.com/gradle/gradle-enterprise-api-samples/blob/main/build.gradle.kts#L18).